client/src/App.css

```diff
@@ -0,0 +1,40 @@
+.top-nav-bar {
```

Comment:
Pretty sure this file was just generated from the dependencies ryan added last week when i did npm install.

Reply:
This should be a .scss file now, not .css

Reply:
@rgreen312 so wait should i rename this file? or has the other one already been created and this got added back because of some weird git stuff?